docs/development/Crafting-System.md

```diff
@@ -0,0 +1,90 @@
+# Crafting System
+
+This page will help you to create/remove recipes/categories. Any detailed information about the system available in the system's documentation (in the code) or Unity Editor's tooltips. But you can ask any question about the system in Discord - just ping Lizurt#4269.
+
+## How to...
+
+### How to create a new crafting recipe
+
+1. Open the Unity Editor 
+2. Follow the path: /Assets/ScriptableObjects/Crafting/Recipes/
+3. Find a necessarry category to put a new recipe into.
+4. RMB -> Create -> ScriptableObjects -> Crafting -> CraftingRecipe
+5. Give the new recipe a name. 
+6. Press the "Add to the singleton if necessary" button.
+7. Edit your new recipe freely(tooltips will help you).
+8. Add the opportunity for a player to learn this recipe.
+  * You can add the recipe to the Player's default known recipes list in the PlayerCrafting component.
```

Comment:
Might want to specify where the `PlayerCrafting` component is.

Though, discussion in Discord suggests moving this into an SO for creating recipe collections, and then having a list of these recipe collections defined on the occupation SOs, at some point. Something like that.

Reply:
> Might want to specify where the PlayerCrafting component is.

Well it's already specified or something like that:

> ... to the Player's ...

Do you mean it's better to write `... to the Player_V4's ...`?

Reply:
Yeah, that it is on the player prefab.

Reply:
> Though, discussion in Discord suggests moving this into an SO for creating recipe collections, and then having a list of these recipe collections defined on the occupation SOs, at some point. Something like that.

I think we may edit the documentation when we implement this.

Reply:
Yep, outside the scope of this one.